posts/blender_w3d_plugin/blender_w3d_plugin.md

```diff
@@ -0,0 +1,64 @@
+
+You probably already know that the Westwood 3D (W3D) format is the 3D format used in the first games that were created with the SAGE engine (later on replaced by a serialized version called W3X). One big issue with that format used to be that there are very little tools that allow authoring of these files. The offical modding SDKs contained plugins for old Autodesk 3dsmax versions (which were recent at the time). The issue is that Autodesk 3dsmax is a commercial tool and the offical plugin is not available for current versions anymore. Apart from this it was lacking some features the community desperatly needs, like the import of compressed animations.
+
+As a conclusion of this we decided to write an opensource Blender Plugin, which allows to export and import W3D files. Our existing W3D code from OpenSage was a big advantage, since it helped us getting a strong understanding of the format itself.
+
+# Why Blender
+
+With the latest major Blender 2.8 release it became the first choice for opensource modelling software. Big companies like Epic, NVIDIA, AMD and Ubisoft have decided to fund the development of Blender. Due to the open Python API documentation it's also easy to write plugins that interact with the software.
+
+![Blender Logo](./blender_logo.png)
+
+# How it works
+
+*This walkthrough assumes you know how Blender works and how to use it*
+
+As a user of the plugin you just want to install the latest version of the plugin, which can be found [here](https://github.com/OpenSAGE/OpenSAGE.BlenderPlugin/releases). Through the preferences window you can select that zip archive to be installed. The import and export submenus now contain entries for the `*.w3d` format.
+During import of a model it's important that all referenced files (Skeleton & Textures) can be found, so they must be in the same folder as the file you want to import.
+
+## General
+
+So at first we'll start by importing a simple structure, like the Gondor Barracks from Battle for Middleearth:
+![Gondor Barracks](./barracks.png)
+
+The model as you can see it in the screenshot above is in it's default state. As you can see in the scene collection on the right we create a parent collection for all meshes that are part of the `W3D` file. Since materials in Blender 2.8 are completly node based the plugin automatically creates a node based shader setup that matches the material specified inside the model. See below for the barracks shader node setup, which uses a diffuse and normal texture.
+
+![Gondor Barracks Shader](./barracks_nodes.png)
+
+Since there are many material options, which are specific to `W3D` we added two new property groups to the material section of each object. This groups are required to set advanced options of the Westwood3D format, which are quite specific to the SAGE game engine. So these options are only used for export again and have no effect in Blender itself.
```

Comment:
```suggestion
Since there are many material options which are specific to `W3D`, we added two new property groups to the material section of each object. These groups are required to set advanced options of the Westwood3D format, which are quite specific to the SAGE game engine. So these options are only used for the exported file and have no effect in Blender itself.
```